met-web/src/helper/index.ts

```diff
@@ -2,4 +2,15 @@ const replaceUrl = (URL: string, key: string, value: string) => {
     return URL.replace(key, value);
 };
 
-export { replaceUrl };
+interface Params {
+    [param: string]: string;
+}
+const replaceAllInURL = ({ URL, params }: { URL: string; params: Params }) => {
+    const regex = new RegExp(Object.keys(params).join('|'), 'gi');
+
+    return URL.replace(regex, function (matched) {
+        return params[matched];
+    });
+};
+
```

Comment:
it's just my cynical mind. does it save a lot of code ? can we achieve this by normal variable replacement or something like that?
asking bcoz we dont have any test to cover this method and if some point , some bug happens, will be hard to debug this..
if we have a strong use case ;like replacing lot of variables in url always , i am all in for it..
but just worried about maintainability...if its just replacing our params in url , may be simple variable substitions..
lol..definitly point of contention..

Reply:
Some URLs may have multiple path variables, I added the above to handle survey/survey_id/engagement/engagement_id. I I thought we might need a method to handle that in one go. Researched and took it from here https://stackoverflow.com/questions/15604140/replace-multiple-strings-with-multiple-other-strings. 
What do you think? Not necessary?
I thought if the helper method itself is ugly, it would at least be hidden and it's calling is clear and easy.